docs/style/entities/author.md

```diff
@@ -60,144 +61,156 @@ should be related to
 ## Author Credits
 This section explains about author credits.
 
-When entering books into BookBrainz, it's important to understand the difference between the concept of an <u>Edition</u>, the physical item you can hold in your hands, and the literary concept of <u>Works</u> that books contain. (see also [Wikipedia](https://en.wikipedia.org/wiki/Creative_work) for more information.)<br>
+When entering books into BookBrainz, it's important to understand the difference between the concept of an <u>Edition</u>, the physical item you can hold in your hands, and the literary concept of <u>Works</u> that books contain. (see also [Wikipedia](https://en.wikipedia.org/wiki/Creative_work) for more information.)
```

Comment:
This seems strange. It says it's important to understand the difference, but it doesn't actually do much about explaining the difference and neither does it link to a better explanation?